style.css

```diff
@@ -46,8 +50,29 @@ body {
 .contact-me .section-content input {
   width: 100%;
 }
-.contact-me .section-content input[name=message] {
+.contact-me .section-content textarea[name=message] {
   height: 200px;
+  width: 100%;
+}
+.contact-me .section-content .submit-button,
+.download a,
+.github button,
+.share-options {
+  background-color: white;
+  border: 2px solid #c3f2e5;
```

Comment:
did you try using css variables to store the colors? it is really helpful, if you in the future would like to change it, you will just need to change the variable instead of tons lines of codes